cpg_workflows/workflow.py

```diff
@@ -467,15 +461,6 @@ def expected_outputs(self, target: TargetT) -> ExpectedResultT:
         Can be a str, a Path object, or a dictionary of str/Path objects.
         """
 
-    def deprecated_queue_for_cohort(self, cohort: Cohort) -> dict[str, StageOutput | None]:
```

Comment:
There isn't a single instance of `deprecated_queue_for_cohort` being called in this whole codebase, and we're pretty far into the MultiCohort changes, so instead of renovating these methods we should delete them.

Reply:
Funnily, we made this call in a separate PR around the same time.